src/modules/helpers/throwError.ts

```diff
@@ -0,0 +1,11 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export const throwError = (status: HttpStatus, error: string) => {
+  throw new HttpException(
```

Comment:
@gerimujo496 Low priority: Looks redundant, you could simply throw the HTTPError inline